docs/v0.4/en/reference/data-types.md

```diff
@@ -47,6 +47,44 @@ The following table lists the corresponding alias.
 |TimestampMicrosecond| Timestamp_us , Timestamp(6)|
 |TimestampNanosecond|Timestamp_ns , Timestamp(9)|
 
+## Decimal
+
+GreptimeDB supports `decimal` type, which is a fixed-point type. 
+
+It is represented as `decimal(precision, scale)`, where `precision` is the total number of digits and `scale` is the 
+number of digits in the fractional part. For example, `123.45` has a precision of 5 and a scale of 2.
+
+> ⚠️：It is an undefined behavior if we use `Decimal(3, 2)` to store `123.45`.
```

Comment:
why do we emphasize this? If the inserted data does not comply with the decimal rules, an error should be thrown.